ocamldoc/odoc_html.ml

```diff
@@ -2419,7 +2419,9 @@ class html =
         let f_ele e =
           let simple_name = Name.simple (name e) in
           let father_name = Name.father (name e) in
-          if strip_stdlib && father_name = "Stdlib" && father_name <> simple_name then
+          if strip_libname &&
+               father_name = !Odoc_global.library_namespace &&
```

Comment:
I forgot that this need to be guarded by `father_name <> ""` otherwise not-setting `-lib` will remove any toplevel modules.

Reply:
Fixed